live/global.ini

```
@@ -63842,43 +63842,43 @@ planetcollect_illegalwaste_desc_0003=Ich habe gerade eine dringende Nachricht vo
 planetcollect_illegalwaste_desc_0004=Ich habe eine Anfrage für eine Reinigungskraft erhalten. Da ist ein Paket eingepackt und bereit, von der Welt weggeschleppt zu werden. Es braucht nur noch ein Schiff, das es an einen Ort bringt, wo die Sicherheitskräfte nicht hinschauen. \n\n~mission(Stops)Und denken Sie nicht einmal daran, sie zu öffnen. Was auch immer drin ist, sollte besser verschlossen bleiben. ~mission(Timed)
 planetcollect_illegalwaste_desc_0005=Der beste Teil unserer Arbeit? Man weiß nie, was man von einem Tag auf den anderen tun wird. \n\nNehmen Sie zum Beispiel heute. Heute werden Sie ein Paket von der Welt verschwinden lassen. ~mission(Stops)~mission(Timed)\n\nWie wäre es damit? Wer weiß, was du morgen machen wirst, oder?
 planetcollect_illegalwaste_desc_0006=Du musst ~mission(FakeOut)... Ha! Wäre das nicht großartig? \n\nSpaß beiseite, du musst dir ein Paket schnappen und es außerhalb des Planeten entsorgen. ~mission(Timed) ~mission(Stops)Danke.
-planetcollect_illegalwaste_fakeout_0001=raub die Prime Central Core Bank aus
+planetcollect_illegalwaste_fakeout_0001=raube die Prime Central Core Bank aus
```

Wenn man das mit dem drüber stehenden Satz kombiniert kommt nur 💩 raus:
```suggestion
planetcollect_illegalwaste_fakeout_0001=die Prime Central Core Bank ausrauben
```

---

Vlt muss man die desc Sätze anpassen damit die folgenden \_fakeout\_ alle grammatisch korrekt sind.